charger/viessmann/identity.go

```diff
@@ -0,0 +1,170 @@
+package viessmann
+
+import (
+	"context"
+	"fmt"
+  "os"
```

Comment:
Du brauchst gofmt oder einfach VScode

Reply:
Die Datei selbst muss nach plugin/auth

Reply:
Ich verstehe nicht ganz - soll die generisch werden und als `plugin/auth/identity.go` dort liegen, in der Hoffnung dass alle oauth2 provider identisch sind? Oder Viessmann-spezifisch als `plugin/auth/viessmann.go`?

Reply:
Letzteres. Deshalb muss sie sich mit eigenem Namen registrieren.